NEWS.md

```diff
@@ -82,6 +82,7 @@ New library functions
   efficiently ([#35816]).
 * New function `addenv` for adding environment mappings into a `Cmd` object, returning the new `Cmd` object.
 * New function `insorted` for determining whether an element is in a sorted collection or not ([#37490]).
+* New function `Base.indent` for indenting non-blank lines by a number of spaces.
```

Comment:
I guess this should either be exported or not be mentioned here?

Reply:
As `unindent` isn't exported I'll also leave this not exported. Is there a place in news to put non-exported functions?

Reply:
This is meant to be public API right? Then it needs to be added to the manual.

Reply:
Functions that are not used in `Base` and that are not public API are essentially dead code. It is a bit strange to implement something like this without either having a usecase for it inside `Base`, or advertise it publicly (e.g. document properly and export it).